kolena/_utils/batched_load.py

```diff
@@ -79,6 +81,26 @@ def upload_data_frame_chunk(df_chunk: pd.DataFrame, load_uuid: str) -> None:
 DFType = TypeVar("DFType", bound=LoadableDataFrame)
 
 
+def upload_data_frame(df: pd.DataFrame, load_uuid: str, rows: int = 1000) -> None:
+    df_memory = df.memory_usage(index=True, deep=True).sum()
+    batch_size = (
```

Comment:
Can we add unit test to verify the chunking logic, maybe with some mock and reduced size?

Reply:
Will add

Reply:
added unit tests